polaris-tokens/scripts/toJSON.ts

```diff
@@ -1,37 +1,26 @@
 import fs from 'fs';
 import path from 'path';
 
-import {Tokens, TokenProperties} from '../src';
+import {MetaTokens, TokenGroup} from '../src';
 
 const outputDir = path.join(__dirname, '../dist/json');
 
 function getFileName(fileName: string) {
   return path.join(outputDir, fileName);
 }
 
-export async function toJSON(tokens: Tokens) {
+export async function toJSON(tokens: MetaTokens) {
   if (!fs.existsSync(outputDir)) {
     await fs.promises.mkdir(outputDir, {recursive: true});
   }
 
   for (const entry of Object.entries(tokens)) {
-    const [tokenGroup, tokenProps] = entry as [string, TokenProperties];
+    const [tokenGroupName, tokenGroup] = entry as [
+      keyof MetaTokens,
+      TokenGroup,
+    ];
+    const fileName = getFileName(`${tokenGroupName}.json`);
```

Comment:
```suggestion
    const filePath = path.join(outputDir, `${tokenGroupName}.json`);
```

**thought:** `getFileName` is kind of an unnecessary abstraction now, what do you think?